src/main/resources/org/sonar/plugins/checkstyle/metadata/additional-rules-meta.yml

```diff
@@ -0,0 +1,4 @@
+rules:
+    - rule: EmptyBlockCheck
+    - rule: JavadocPackageCheck
+      tag: comment
```

Comment:
please put here Indentation only.

and name it `rules-meta.yml`, it is not additional, it is override

Reply:
Done